Component versions (#86)
mulesoft/docs-private-cloud: modules/ROOT/pages/install-workflow.adoc

include::_attributes.adoc[]
endif::[]

> After your infrastructure team verifies that the required infrastructure has been correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites], you can install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) using one of the following installers: 

Reviewer: ```suggestion
After your infrastructure team verifies that the required infrastructure is correctly installed and configured as described in xref:install-checklist.adoc[Product Prerequisites], you can install Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) using one of the following installers: 
```
